### PR TITLE
Update new env wrapper

### PR DESCRIPTION
**Pull request by limberc**
## Update new env wrapper for RL env.

Add DeepMind Env and several Atari Env wrapper from https://github.com/facebookresearch/torchbeast/blob/3f3029cf3d6d488b8b8f952964795f451a49048f/torchbeast/atari_wrappers.py

### CODE REVIEW

**Comment by limberc** [comment]
Seems that the supporting procgen is a little bit messy.

I would try to support procgen later.

**Comment by limberc** [comment]
Hi there, is there anything wrong here?

**Review by Borda — APPROVED** [review]

File: tests/models/rl/unit/test_wrappers.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from unittest import TestCase
 
 import gym
+import numpy as np
+import torch
```
Comment (Borda) [inline tests/models/rl/unit/test_wrappers.py:5]:
```suggestion
```

File: pl_bolts/models/rl/common/gym_wrappers.py [hunk 2]
```diff
@@ -1,19 +1,23 @@
 """
 Set of wrapper functions for gym environments taken from
 https://github.com/Shmuma/ptan/blob/master/ptan/common/wrappers.py
+and
+    https://raw.githubusercontent.com/openai/baselines/7c520852d9cf4eaaad326a3d548efc915dc60c10/baselines/common/atari_wrappers.py
+
```
Comment (akihironitta) [inline pl_bolts/models/rl/common/gym_wrappers.py:6]:
```suggestion
https://github.com/openai/baselines/blob/7c520852d9cf4eaaad326a3d548efc915dc60c10/baselines/common/atari_wrappers.py
```

**Comment by Borda** [comment]
@limberc could you please resolve the conflict so we can quickly merge it... :rabbit:

**Pull request closed without merging**